third_party/move/move-vm/transactional-tests/tests/tests.rs

```diff
@@ -87,6 +87,11 @@ fn vm_config_for_tests(verifier_config: VerifierConfig) -> VMConfig {
 /// "foo" will have a separate baseline output file `test.foo.exp`.
 const SEPARATE_BASELINE: &[&str] = &["/function_values_safety/"];
 
+/// Test files which should echo their commands. For now we only do this with function
+/// values, as those use macros for clarity what is executed. Perhaps it should be done
+/// for all tests.
```

Comment:
I think `echo` is very useful when going over tests, we should turn it on for all, but in another PR